src/applications/my-education-benefits/config/form.js

```diff
@@ -796,6 +796,19 @@ const formConfig = {
                 'ui:title':
                   'Would you like to receive text message notifications on your education benefits?',
                 'ui:widget': 'radio',
+                'ui:validations': [
+                  (errors, field, formData) => {
+                    if (
+                      !formData['view:phoneNumbers'].mobilePhoneNumber.phone ||
+                      formData['view:phoneNumbers'].mobilePhoneNumber
+                        .isInternational
+                    ) {
```

Comment:
The above logic is off. If will error if "No, just send me email notifications is selected" too.

Reply:
Also, you should make the enums more simple (YES and NO, for example, rather than the whole string) and change the [form.receiveTextMessages] section in the uiSchema to have a 
ui:options': {
  labels: {...},
  widgetProps: {...},
  selectedProps: {...},
}

for easier reference to the selected property as well as 508 compliance (with the ARIA attributes). Reference formFields.benefitSelection.

Reply:
thanks for the heads up